tutorcodejail/patches/local-docker-compose-jobs-services

```diff
@@ -1,6 +1,8 @@
-codejail_apparmor-job:
```

Comment:
I tested my local docker installation with this, and it seems to be working.

Reply:
The problem is when using kubernetes. Tutor will fetch jobs with the name `codejail_apparmor` but we can not define such job beacause names with underscores in are not valid in Kubernetes.